src/scripts/lychee.js

```diff
@@ -144,6 +144,12 @@ lychee.logout = function() {
 
 }
 
+lychee.restart = function() {
+
+	document.location.href = ''
```

Comment:
Ah, right. Then I would opt for keeping `location.href = ''`, because that's what we want in this case and it does not depend on other checks afterwards.